Vagrantfile

```diff
@@ -33,9 +33,6 @@ Vagrant.configure(2) do |config|
         vb.cpus = cpus
     end
 
-    # Attach an extra disk to /home/vagrant
-    eval (File.read 'vagrant-cookbook/host-system-config/attach-extra-disk.vagrantfile')
-    config.vm.provision "shell", args: ["/dev/sdb"], path: "vagrant-cookbook/host-system-config/create-disk.sh"
```

Comment:
I do not know. It might be the case that we need to add this again. However, it does not work in this setup since it is hard coded for VBox. So at the very least it needs some overhaul either way.